GLTF_PBR_Renderer/interface/GLTF_PBR_Renderer.hpp

```diff
@@ -109,7 +109,8 @@ class GLTF_PBR_Renderer
     GLTF_PBR_Renderer(IRenderDevice*     pDevice,
                       IRenderStateCache* pStateCache,
                       IDeviceContext*    pCtx,
-                      const CreateInfo&  CI);
+                      const CreateInfo&  CI,
+                      int                numBRDFSamples = 512);
```

Comment:
I suggest adding this parameter to the `CreateInfo` struct.
Please use naming style consistent with the rest of the API: `NumBRDFSamples`
Please use `Uint32` type

Reply:
Done as requested!